frontend_app_folder_advanced/test/container/FolderAdvanced.spec.js

```diff
@@ -73,26 +80,46 @@ describe('<FolderAdvanced />', () => {
 
       describe('handleFolderChanged (different folder)', () => {
         const tlmData = {
+          author: {
+            avatar_url: null,
+            public_name: 'Global manager',
+            user_id: 1
+          },
           content: {
             ...folder,
             content_id: 2
           }
         }
 
-        const wrapper = shallow(<FolderAdvanced {...props} />)
+        // const wrapper = shallow(<FolderAdvanced {...props} />)
```

Comment:
if you don't use this line anymore, feel free to remove it

Reply:
Changed